### PR TITLE
Update lost_or_stolen.md

### PR DESCRIPTION
**Pull request by EThom73**
**What:**
Important updates to the Lost and Stolen guidance in line with ISO 27001  
**Why**
This covers our obligation to document and make available to the team our guidance around device security, and the process to follow if your device is lost or stolen
**Notes:**
The links are to documents that don't exist/exist only in draft. They will be updated once the structure of the IMS file is established

### CODE REVIEW

File: ims/company/lost_or_stolen.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
+
+For Team members using the standard build of Made Tech device, we are able to use MDM software to remotely wipe data from your device.
```
Comment (lukemorton) [inline ims/company/lost_or_stolen.md]: We should explain acronyms or link to definitions, methinks.

File: ims/company/lost_or_stolen.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
+
+For Team members using the standard build of Made Tech device, we are able to use MDM software to remotely wipe data from your device. 
+
+Those who are not using a Made Tech issued device should consult the [BYOD policy.](byod.md)
```
Comment (lukemorton) [inline ims/company/lost_or_stolen.md]: A very very very minor point, excuse my pedanticness, but I'd move the full stop outside of the link.

File: ims/company/lost_or_stolen.md (hunk 1 above)
Comment (EThom73) [reply ims/company/lost_or_stolen.md]: MDM has an explanation. 

IMS manager and CSO have been explained elsewhere in the IMS File

File: ims/company/lost_or_stolen.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
```
Comment (lukemorton) [inline ims/company/lost_or_stolen.md]: We should link to an explanation of CSO and IMS Manager as not everyone will find this page from the IMS README.

**Pull request closed without merging**